aarch64_linux/aarch64_wheel_ci_build.py

```diff
@@ -86,7 +86,10 @@ def update_wheel(wheel_path) -> None:
         "/usr/local/cuda/lib64/libcudnn_engines_precompiled.so.9",
         "/usr/local/cuda/lib64/libcudnn_heuristic.so.9",
         "/opt/conda/envs/aarch64_env/lib/libgomp.so.1",
-        "/opt/OpenBLAS/lib/libopenblas.so.0",
```

Comment:
Is this file used for CUDA-only build? My gut feels like removing `/opt/OpenBLAS/lib/libopenblas.so.0` here will cause missing libraries in CPU-only build.

Reply:
thanks for catching this, let me add the guards to here as well